src/System.Security.Cryptography.Cng/tests/Configurations.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netfx-Windows_NT;
```

Comment:
oh maybe that's the problem then, it can't find the types defined on it's own contract, which today, the ref builds for platform-specific since it used to be netstandard. It makes sense that if the types were removed from standard that we would have started to see this problem, in that case, is the right fix to make Cng ref and impl to be netstandard?